config/templates/cluster.yaml

```diff
@@ -165,8 +165,12 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # experimental:
 #   nodeDrainer:
 #     enabled: true
+#   awsEnvironment:
+#     enabled: true
+#     environment:
+#       CFNSTACK: '{ "Ref" : "AWS::StackId" }'
```

Comment:
I'm putting it under experimental because it's very easy for users to break their template here. Was debating myself whether to further spec it into function & parameters but i think this would be way too opaque even if we could do some better syntax checking that way.